packages/core/src/mappers/renderer.ts

```diff
@@ -186,7 +186,7 @@ export const doCreateDefaultValue = (
   rootSchema: JsonSchema
 ) => {
   const resolvedSchema =
-    typeof schema.$ref === 'string'
+    typeof schema?.$ref === 'string'
```

Comment:
schema should not be `undefined` here. If it is. then we should adjust the parameter type

Reply:
fixed